runtime/common/src/changes.rs

```diff
@@ -44,18 +55,17 @@ impl<T: Changeable, Options: Clone> RuntimeChange<T, Options> {
 				LoansChange::<T>::Policy(_) => vec![week, blocked],
 				LoansChange::<T>::TransferDebt(_, _, _, _) => vec![],
 			},
+			RuntimeChange::OracleCollection(change) => match change {
+				OracleCollectionChange::Feeders(_, _) => vec![], // TODO: specify
```

Comment:
Reminder for this open `TODO`

Reply:
I think it's ok for now to leave it empty, WDYT @mustermeiszer ?